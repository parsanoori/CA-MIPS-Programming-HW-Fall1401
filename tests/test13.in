7
793035292
162332284
1000494823
358470057
19848823
410124970
734371151