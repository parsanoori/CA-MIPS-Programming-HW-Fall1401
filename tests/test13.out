19848823
162332284
358470057
410124970
734371151
793035292
1000494823